src/pg/sql/getMeta.js

```diff
@@ -0,0 +1,15 @@
+import sql, { join, raw } from 'sql-template-tag';
+import { getJsonBuildObject, nowTimestamp } from './helper';
+
+export const getIdMeta = ({ idCol }) =>
+  getJsonBuildObject({
+    $key: sql`"${raw(idCol)}"`,
+    $ver: nowTimestamp,
+  });
+
+export const getMeta = (key, prefix, idCol) =>
```

Comment:
This is great! Minor point, let's call it `getArgMeta`, to be consistent with all other read code where we have two branches, `id` and `arg`.